src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeCustomAttributeData.cs

```diff
@@ -1794,8 +1830,26 @@ internal static AttributeUsageAttribute GetAttributeUsage(RuntimeType decoratedA
                 if (attributeUsageAttribute is not null)
                     throw new FormatException(SR.Format(SR.Format_AttributeUsage, attributeType));
 
-                ParseAttributeUsageAttribute(caRecord.blob, out AttributeTargets targets, out bool inherited, out bool allowMultiple);
-                attributeUsageAttribute = new AttributeUsageAttribute(targets, allowMultiple, inherited);
+                CustomAttributeEncodedArgument.ParseAttributeArguments(
+                    caRecord.blob,
+                    ctorParams,
+                    namedParams,
+                    (RuntimeModule)typeof(AttributeUsageAttribute).Module);
+
+                // Convert parsed attribute arguments
+                AttributeTargets attrTargets = ctorParams[0].EncodedArgument is not null
```

Comment:
```suggestion
                AttributeTargets attrTargets = ctorParams[0].EncodedArgument?.PrimitiveValue.Byte4 ?? 0;
```
Would something like this work to avoid `!` nullability suppression and the repetition?

Reply:
I really don't like the inline `?`. I don't find it helpful and in fact rather too subtle about what is going on. I'm fine doing it if that is preferred, but to me that feature is merely a copy/paste footgun for introducing unnecessary checks or logic that isn't intended.

Reply:
My comment was triggered by use of `!` to suppress nullability warning. Suppressed nullability warnings are hiding potential bugs.

If you do not like `?.`, store the intermediate result in a local variable to avoid `!` for nullability warning suppression.
```
CustomAttributeEncodedArgument? encodedArgument = ctorParams[0].EncodedArgument;
 AttributeTargets attrTargets = (encodedArgument != null) ? encodedArgument.PrimitiveValue.Byte4 : 0;
```

Reply:
Updated.